MAUI/ProgressBar/Appearance.md

```diff
@@ -0,0 +1,392 @@
+* `ThicknessUnit`: Specifies whether the `ProgressThickness` or `TrackThickness` are defined in pixel or factor.
+
+The `ProgressThickness` or `TrackThickness` of the progress bar can be specified either in pixel or factor. If the `ThicknessUnit` is specified as Pixel, then the range will be rendered based on the provided pixel value. If the `ThicknessUnit` is set as factor, the provided factor value will be multiplied with outer radius. For example, if the thickness width is set as 0.1, then 10% of outer radius is considered as thickness.
+
+The following code sample demonstrates how to customize the appearance of circular progress bar.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<!--Circular progress bar with radius customization -->
+
+<progressBar:SfCircularProgressBar Progress="75"
+                                   TrackRadiusFactor="0.8" 
+                                   ProgressRadiusFactor="0.75"
+                                   ThicknessUnit="Factor"
+                                   TrackThickness="0.05"
+                                   ProgressThickness="0.05">
+</progressBar:SfCircularProgressBar>   
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar();
+circularProgressBar.Progress = 75;
+circularProgressBar.TrackRadiusFactor = 0.8;
+circularProgressBar.ProgressRadiusFactor = 0.75;
+circularProgressBar.TrackThickness = 0.05;
+circularProgressBar.ProgressThickness = 0.05;
+circularProgressBar.ThicknessUnit = SizeUnit.Factor;
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![appearance](images/appearance/circular-thickness.png)
+
+## Corner radius
+
+The `CornerRadius` property is used to customize the rounded edges in the linear progress bar as demonstrated in the following code sample.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<progressBar:SfLinearProgressBar Progress="50"
+                                 TrackHeight="10" 
+                                 TrackCornerRadius="5"
+                                 ProgressHeight="10"
+                                 ProgressCornerRadius="5">
+    
+</progressBar:SfLinearProgressBar>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar();
+linearProgressBar.Progress = 50;
+linearProgressBar.TrackHeight = 10;
+linearProgressBar.ProgressHeight = 10;
+linearProgressBar.ProgressCornerRadius = 5;
+linearProgressBar.TrackCornerRadius = 5;
+this.Content = linearProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+![cornerradius](images/appearance/corner-radius.png)
+
+## Corner style customization
+
+The `ProgressCornerStyle` and `TrackCornerStyle` property of circular progress bar specifies the corner type for the progress and track. The corners can be customized using the `BothFlat`, `BothCurve`, `StartCurve`, and `EndCurve` options. The default value of this property is `BothFlat`.
```

Comment:
The `ProgressCornerStyle` and `TrackCornerStyle` property of the circular progress bar specifies the corner type for the progress and track. The corners can be customized using the `BothFlat,` `BothCurve,` `StartCurve,` and `EndCurve` options. The default value of this property is `BothFlat.`

Reply:
done